src/plugins/discover/public/application/angular/directives/histogram.tsx

```diff
@@ -289,7 +289,6 @@ export class DiscoverHistogram extends Component<DiscoverHistogramProps, Discove
       headerFormatter: this.renderBarTooltip(xInterval, domainStart, domainEnd),
       type: TooltipType.VerticalCursor,
     };
-
```

Comment:
This change is triggering the KibanaApp review, does it really make sense?

Reply:
Sorry for that, I've created another PR to stop this.